bin/optimize.js

```diff
@@ -20,17 +25,49 @@ let options = {
   multipass: true
 };
 
+/**
+ * Reads an icon file off disk and optimizes it, saving to same location
+ * @param {array}              filePaths  array of relative file paths
+ * @param {SVGO}               svgo       SVGO instance with correct options
+ * @param {SingleBar}          bar        progress bar instance
+ * @return {promise}
```

Comment:
I believe Promise has to be capitalized (see https://jsdoc.app/tags-returns.html).